src/lib/datepicker/month-view.ts

```diff
@@ -67,6 +67,9 @@ export class MdMonthView<D> implements AfterContentInit {
   /** Emits when a new date is selected. */
   @Output() selectedChange = new EventEmitter<D | null>();
 
+  /** Emits when date is picked */
+  @Output() userSelection = new EventEmitter();
```

Comment:
The comment you had above seems more appropriate for this as well:

```
/** Emits when any date is selected. */
```

Nit: Add a period to both to match other comments